lib/customers.rb

```diff
@@ -0,0 +1,12 @@
+class Customers
+    attr_accessor :a_customer
+    
+    def initialize
+        @a_customer = {}
```

Comment:
Does the curly brackets mean you are creating a hash? I always forget this syntax and use [ ] for an array. Thank you for the reminder that { } is an option.